src/PeopleSDKAdapter.js

```diff
@@ -0,0 +1,112 @@
+import {from, fromEvent, concat} from 'rxjs';
+import {base64} from '@webex/common';
+import {PeopleAdapter, PersonStatus} from '@webex/component-adapter-interfaces';
+import {finalize, publish, refCount, flatMap, map, filter} from 'rxjs/operators';
+
+const USER_PRESENCE_UPDATE_EVENT = 'event:apheleia.subscription_update';
+
+/**
+ * The `PeopleSDKAdapter` is an implementation of the `PeopleAdapter` interface.
+ * This adapter utilizes the Webex JS SDK to fetch data about a person.
+ *
+ * @class PeopleSDKAdapter
+ * @extends {PeopleAdapter}
+ */
+export default class PeopleSDKAdapter extends PeopleAdapter {
+  constructor(datasource) {
+    super(datasource);
+
+    this.getPersonObservables = {};
+  }
+
+  /**
+   * Returns a proper Enum key for a given PersonStatus value
+   *
+   * @param {string} status
+   * @returns {string} PersonStatus
+   * @memberof PeopleSDKAdapter
+   */
+  getStatus(status) {
+    return Object.keys(PersonStatus).find((key) => PersonStatus[key] === status);
+  }
+
+  /**
+   * Returns the UUID of a person from the given public ID
+   *
+   * @param {string} ID
+   * @returns {string}
+   * @memberof PeopleSDKAdapter
+   */
+  getPersonUUID(ID) {
+    return base64.decode(ID).split('/')[4]; // ciscospark://us/PEOPLE/<UUID>
+  }
+
+  /**
+   * Fetches the person data from the sdk and returns in the shape required by adapter.
+   *
+   * @param {string} ID
+   * @returns {Person}
+   * @memberof PeopleSDKAdapter
+   */
+  async fetchPerson(ID) {
+    const {id, emails, displayName, firstName, lastName, avatar, orgId} = await this.datasource.people.get(ID);
+
+    return {
+      ID: id,
+      emails,
+      displayName,
+      firstName,
+      lastName,
+      avatar,
+      orgID: orgId,
+    };
+  }
+
+  /**
+   * Returns an observable that emits person data of the given ID.
+   *
+   * @param {string} ID ID of person to get.
+   * @returns {Observable.<Person>}
+   * @memberof PeopleSDKAdapter
+   */
+  getPerson(ID) {
+    if (!(ID in this.getPersonObservables)) {
+      const personObs = from(this.fetchPerson(ID));
```

Comment:
Can we just call this `person`? Not sure I personally like the `obs` prefix, i think it makes the readability flow a little harder (See my other comment on line 81

Maybe adopt [Angular's naming convention](https://angular.io/guide/rx-library#naming-conventions-for-observables)?

Reply:
I like the angular naming convention of putting the `$` at the end. I was wondering about that when reading through all the docs and example code.